docker-compose/README.md

```diff
@@ -0,0 +1,4 @@
+## Instructions
```

Comment:
Can you just elaborate on why this exists in this repository? "To test locally HA..."

Reply:
why not? I think users should now that they must change license info

Reply:
Haha, we have a misunderstanding, I was referring to elaborate in README.md why the docker-compose exists and what is the idea behind it :D